src/controller/data/System/SystemConfiguration.cs

```diff
@@ -7,30 +7,69 @@ public class SystemConfiguration
 {
     public int Id { get; set; }
     public string SystemName { get; set; } = "Controller";
-    public TimeZones TimeZone { get; set; } = TimeZones.UTC;
+    public string TimeZone { get; set; } = TimeZoneList.TimeZones.Select(s => s.Code).First(s => s == "UTC");
     public int TimeBetweenDataUpdatesSeconds { get; set; } = 5;
 }
 
-public enum TimeZones // TODO add more time zones
+public class TimeZone()
 {
-    UTC = 0,
-    CET = 1,
-    EET = 2,
-    EST = -5,
-    CST = -6,
-    MST = -7,
-    PST = -8,
-    GMT = 0,
-    IST = 5,
-    JST = 9,
-    AEST = 10
+    public string Code { get; set; }
+    public double DifferenceToUTC { get; set; }
+}
+
+public static class TimeZoneList
+{
+    public static List<TimeZone> TimeZones = new List<TimeZone>()
```

Comment:
And just put this in TimeZone class as static field and move to separate file pls for bigger blocks